src/plugins/msigdb/upload.py

```diff
@@ -16,7 +16,8 @@ class msigdbUploader(uploader.BaseSourceUploader):
         "src_meta": {
             'license_url': 'https://www.gsea-msigdb.org/gsea/msigdb_license_terms.jsp',
             'licence': 'CC Attribution 4.0 International',
```

Comment:
use the short name from https://creativecommons.org/licenses/by/4.0/ as "CC BY 4.0"